### PR TITLE
returnAllOrganisms parameter for add_organism

### PR DESCRIPTION
**Pull request by MoffMade**
I added the parameter in `apollo/organisms /__init__.py` and ran `make rebuild` but it only seemed to change the docs, not the arrow files. I can update them manually if needed, I'm not sure the next step there.

### CODE REVIEW

**Comment by nathandunn** [comment]
@MoffMade  should be `pip install . `

File: apollo/organisms/__init__.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ class OrganismsClient(Client):
 
     @raise_error_decorator
     def add_organism(self, common_name, directory, blatdb=None, genus=None,
-                     species=None, public=False, metadata=None):
+                     species=None, public=False, metadata=None, return_all=None):
```
Comment (hexylena) [inline apollo/organisms/__init__.py:15]: This should be True, not None

File: README.rst [hunk 2]
```diff
@@ -85,6 +85,8 @@ Or with the Arrow client:
 History
 -------
 
+- 4.2.6
+    - Prevent from displaying login/password in the logs
```
Comment (hexylena) [inline README.rst:89]: we should go back and fix this

Comment (abretaud) [reply README.rst:89]: what do you mean? if it's just the changelog, I updated it in #43

File: apollo/organisms/__init__.py (hunk 1 above)
Comment (abretaud) [reply apollo/organisms/__init__.py:15]: This one was merged, but I can make another PR for that

File: README.rst (hunk 2 above)
Comment (hexylena) [reply README.rst:89]: Perfect!

File: apollo/organisms/__init__.py (hunk 1 above)
Comment (abretaud) [reply apollo/organisms/__init__.py:15]: Oh it was replaced by `suppress_output=False` in 4.2.7

Comment (hexylena) [reply apollo/organisms/__init__.py:15]: ahh ok. cool, just caught up on a too-old pr then.

Comment (abretaud) [reply apollo/organisms/__init__.py:15]: no problem :)

**Pull request merged**